src/javascript/Create/CreateNewContentAction/createNewContent.utits.js

```diff
@@ -0,0 +1,51 @@
+import {getTreeOfContent} from '~/Create/CreateNewContentAction/CreateNewContent.gql-queries';
+
+export async function getActions(context, variables) {
+    const {data} = await context.client.query({
+        query: getTreeOfContent,
+        variables
+    });
+    console.log(`query result action ${context.key}`, data);
```

Comment:
No need to log that i think

Reply:
and you are right, log have been removed